packages/interfaces/src/index.ts

```diff
@@ -14,6 +14,8 @@ export type B256Address = string;
 
 export type B256AddressEvm = `0x000000000000000000000000${string}`;
 
+export type Bytes = Uint8Array;
```

Comment:
Ethers v6 no longer exports a `Bytes` type, only `BytesLike` which is a `Uint8Array | string`, so I have introduced this for our own usage.